spec/requests/dashboard_spec.rb

```diff
@@ -89,11 +88,11 @@
 
     it 'allows a user to check before deleting an application' do
       applicant = Applicant.find_by(first_name: "Jane")
-      app = CrimeApplicationPresenter.new(applicant.crime_application)
```

Comment:
I think is best not to couple the request tests to any implementation details in the controller, as we want to just test inputs and outputs. It is fine if the implementation changes some day and the output is no longer "Jane Doe" but, for instance, "Doe, Jane". The test will fail then and alert us something has changed 😛

Reply:
Yep better to not do this for sure 👍